### ISSUE TITLE
Invalid annotations?

### ISSUE THREAD

**Issue by smoretti** [SysBioChalmers/Sco-GEM#111]
Hi

some annotations don't look to be valid in the **Sco-GEM.xml** file:

- ec-code: according to identifiers.org ec-code annotations must contain a single EC number.
  For cases with multiple EC numbers, I think you have to duplicate the annotations.
  Don't know really how to deal with OR
  And not sure what you mean with the comma: "2.7.1.11, 2.7.1.56, 2.7.1.144". AND or OR ?
  e.g. "https://identifiers.org/ec-code/&quot;1.2.4.2 and 1.8.1.4 and 2.3.1.61&quot;"
         "https://identifiers.org/ec-code/1.1.1.1 or 1.1.99.1"
         "https://identifiers.org/ec-code/2.7.1.11, 2.7.1.56, 2.7.1.144"

- kegg.reaction: according to identifiers.org kegg.reaction must contain a single KEGG reaction id.
  I think you have to duplicate the annotations.
  Again not sure what you mean with the comma.
  e.g. "https://identifiers.org/kegg.reaction/[&apos;R02075&apos;, &apos;R08836&apos;]"

- doi/PMID: is not valid according to identifiers.org registry for doi
  Better to use https://registry.identifiers.org/registry/pubmed for PMID
  e.g. "https://identifiers.org/doi/PMID: 16554727" should be https://identifiers.org/pubmed/16554727

**Comment by sulheim** [comment]
Hey! 
Thanks for pointing these errors out. The gene annotation would be the place were the and / or relation is defined, so the ec-numbers should be duplicated annotations (i.e. one annotation per enzyme). The reaction 45DOPA should only be annotated to R08836. I'll add these to #105

**Comment by smoretti** [comment]
Hi
it looks Sco-GEM 1.3.0 (xml version) still contains uncommon annotations:
- Some ec-code annotations are not valid according to the pattern defined in identifiers.org
  ec-code annotations must contain a single EC number.
        e.g. "`https://identifiers.org/ec-code/&quot;1.2.4.2 and 1.8.1.4 and 2.3.1.61&quot;`"
                    "`https://identifiers.org/ec-code/3.4.11.1 or 3.4.11.2`"
         should be
```
                "https://identifiers.org/ec-code/1.2.4.2"
                "https://identifiers.org/ec-code/1.8.1.4"
                "https://identifiers.org/ec-code/2.3.1.61"
                "https://identifiers.org/ec-code/3.4.11.-"
```

- Some kegg.reaction annotations are not valid according to the pattern defined in identifiers.org
   kegg.reaction must contain a single KEGG reaction id.
       e.g. "`https://identifiers.org/kegg.reaction/[&apos;R09768&apos;, &apos;R05645&apos;]`"
        should be (if the comma separator means a logic AND)
```
              "https://identifiers.org/kegg.reaction/R09768"
              "https://identifiers.org/kegg.reaction/R05645"
```
              
              
Also it looks the parameters/fluxes "**minus_inf**" and "**plus_inf**" are defined but never user.
The same for the chemical species "**M_g1p_B**".

**Comment by edkerk** [comment]
Now resolved in #145, let us now if you spot any other issues!
The `minus_inf` and `plus_inf` bounds are added by cobrapy to any SBML model by default.

**Comment by smoretti** [comment]
Thanks
Everything looks fine my side

Best
